src/sentry/static/sentry/app/redux/actions/processAssignment.js

```diff
@@ -0,0 +1,78 @@
+import {t} from 'app/locale';
+import {Client} from 'app/api';
+import {substanceSearchEntriesToggleSelectAll} from 'app/redux/actions/substanceSearchEntry';
+
+import {makeActionCreator} from 'app/redux/actions/shared';
+
+// Assigning substances to processes.
+// TODO-nomerge: Rename actions to processAssignments
+// TODO-nomerge: Use the wizard
+export const PROCESS_ASSIGN_SELECT_PRESET = 'PROCESS_ASSIGN_SELECT_PRESET';
+export const processAssignSelectPreset = makeActionCreator(
+  PROCESS_ASSIGN_SELECT_PRESET,
+  'preset'
+);
+
+export const PROCESS_ASSIGN_SELECT_PROCESS_DEF = 'PROCESS_ASSIGN_SELECT_PROCESS_DEF';
+export const processAssignSelectProcess = makeActionCreator(
+  PROCESS_ASSIGN_SELECT_PRESET,
```

Comment:
I think this row is wrong.

Reply:
Yes, I think there is some bug there. I hate to say it, but I think I'll fix it in a new PR.